azkaban-common/src/main/java/azkaban/event/EventData.java

```diff
@@ -32,6 +34,10 @@ public EventData(final ExecutableNode node) {
     this.executionId = (node.getParentFlow() == null) ? -1: node.getParentFlow().getExecutionId();
   }
 
+  public ExecutableNode getNode() {
```

Comment:
Non blocker: Consider exposing a method that tells if the event is meant for executableFlow. we can use the method instead of the expression`getNode() instanceof ExecutableFlow`.

Reply:
done